cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -369,10 +380,12 @@ Future<Void> rollingUpdateForNewCaKey() {
         }
 
         if (podRollReasons.shouldRestart()) {
+            PemTrustSet pemTrustSet = new PemTrustSet(clusterCa.caCertSecret());
+            PemAuthIdentity pemAuthIdentity = PemAuthIdentity.clusterOperator(coSecret);
```

Comment:
Maybe you can name these based on what they are? E.g. `clusterCaTrustSet` and `coAuthIdentity`? I think that can be in general applied to all the parameters and fields -> they should not just copy the type name but they should explain the expected value. E.g. in the `BrokersInUseCheck` above, you have some clear expectation that should be passed in. And I assume it will be the same in many places below.

Reply:
Good point, I'll review the variable names across the board.

Reply:
I've updated the variable names. I've used `coAuthIdentity` as you suggested and then `clusterCaTrustSet` if the variable is used for both ZooKeeper and Kafka clients, and `kafkaCaTrustSet` and `zkCaTrustSet` for the Kafka and ZooKeeper clients respectively, since it is the trustset for the certificate authority that issued and signed the Kafka/ZooKeeper end entity certificate, which happens to be the cluster CA